runtime/lua/vim/lsp/util.lua

```diff
@@ -873,8 +873,10 @@ function M.fancy_floating_markdown(contents, opts)
   -- This is because the syntax command doesn't accept a target.
   local cwin = vim.api.nvim_get_current_win()
   vim.api.nvim_set_current_win(winnr)
+  api.nvim_win_set_option(winnr, 'conceallevel', 2)
+  api.nvim_win_set_option(winnr, 'concealcursor', 'n')
 
-  vim.cmd("ownsyntax markdown")
+  vim.cmd("ownsyntax lsp_markdown")
```

Comment:
I think we need to do `set filetype=lsp_markdown` -- otherwise we won't call the ftplugin stuff I don't think?

Also, this will let people do nice things like:

```
autocmd Filetype lsp_markdown set my command here
```

etc.

Reply:
You're right, fixed.

Reply:
Is it redundant to have `ownsyntax` if we did `filetype` below? I don't actually know.